README.md

```diff
@@ -147,7 +147,7 @@ There are totally 27 workloads in HiBench. The workloads are divided into 6 cate
     
 ### Supported Hadoop/Spark/Flink/Storm/Gearpump releases: ###
 
-  - Hadoop: Apache Hadoop 2.x, CDH5, HDP
+  - Hadoop: Apache Hadoop 3.x, Apache Hadoop 2.x, CDH5, HDP
```

Comment:
+ `Hadoop 3.0, Hadoop 3.1, Hadoop 3.2` if we have tested?

Reply:
@gczsjdy it's done.